.github/workflows/compile.yml

```diff
@@ -8,6 +8,11 @@ on:
 
 env:
   CONFIGURATION_DIRECTORY: ${{ github.workspace }}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  VCPKG_FEATURE_FLAGS: dependencygraph
```

Comment:
what is `dependencygraph` used for?

Reply:
This is supposed to upload information to github to show deps. https://devblogs.microsoft.com/cppblog/vcpkg-integration-with-the-github-dependency-graph/

Reply:
Awesome 👍